here-naksha-lib-hub/src/main/java/com/here/naksha/lib/hub/NakshaHub.java

```diff
@@ -213,8 +221,8 @@ public NakshaHub(
             return defDbCfg; // return default config from DB
           }
         } catch (NoCursor | NoSuchElementException er) {
-          throw unchecked(new Exception(
-              "Unable to read custom/default config from Admin DB - ResultCursor has no data"));
+          logger.info("Not found any db configuration in '" + NakshaAdminCollection.CONFIGS
+              + "' admin collection");
```

Comment:
After this step, app must have one config to continue with startup. And if config is not available then App should fail. So, I think we should continue with exception, unless I am missing reason for ignoring it now.